lib/main.coffee

```diff
@@ -137,9 +152,14 @@ module.exports =
           return messages.map (message) ->
             startPoint = [message.line - 1, message.column - 1]
             endPoint = [message.line - 1, message.column]
-            return {
+            ret = {
               type: message.type
-              text: message.message
               filePath,
               range: [startPoint, endPoint]
             }
+            if showSource
+              ret.html = '<span class="badge badge-flexible">' + (message.source or 'Unknown') + '</span> '
```

Comment:
That's what I did initially but I figured this should never actually happen if everything is working correctly so it would be better to let the user know that they have showSource on but we don't actually know the source.